server/web/config.go

```diff
@@ -535,7 +535,7 @@ func newBConfig() *Config {
 		EnableErrorsShow:   true,
 		EnableErrorsRender: true,
 		Listen: Listen{
-			Graceful:      false,
+			Graceful:      true,
```

Comment:
This should keep false because it's a break change. Users need to specify using true manually